src/main/java/com/forest/mytopmovies/service/user/impl/TokenServiceImpl.java

```diff
@@ -71,6 +81,10 @@ private Claims getClaims(int expireInSec) {
 
     private Map<String, String> parseClaims(Supplier<Claims> toClaims) {
```

Comment:
Converting claims to a hashmap can be extracted to its own method, in parseClaims, 3 things are happening:
1. Checking that claims isn't null
2. Converting claims to a hashmap
3. Returning an empty hashmap if claims are empty or null
Like this:
```
        return Optional.ofNullable(toClaims.get())
                .map(this::convertClaimsToHashMap)
                .filter(Map::isEmpty)
                .orElse(new HashMap<>());
```
Although it might be an idea to change the return type here to Optional<Map> then deal with the empty/null claim problem where parseClaims is called, this would make for a more flexible design.